includes/3rd-party/divi.php

```diff
@@ -65,3 +65,28 @@ function sensei_fix_divi_theme_builder_and_learning_mode_conflict() {
 
 }
 add_action( 'wp', 'sensei_fix_divi_theme_builder_and_learning_mode_conflict' );
+
+/**
+ * Disable the Divi excerpt logic for the Video learning mode template.
+ *
+ * This fixes an error generated by Divi when the Video learning mode template is used.
+ */
+function sensei_fix_divi_learning_mode_video_template_excerpt() {
```

Comment:
Can we have it? it is worth it?

Reply:
Hey Gabriel, sorry, I'm not sure if I understand the concern. Could you elaborate?